sprint0/sprint0/Collision/LinkBlockCollisionHandler.cs

```diff
@@ -72,7 +72,7 @@ private void HandleMovableBlock(IPlayer link, IBlock block, Direction side)
                     block.Location = new Rectangle(block.Location.X, (int)link.Pos.Y + linkSize, block.Location.Width, block.Location.Height);
                     break;
                 case Direction.e:
-                    block.Location = new Rectangle((int)link.Pos.X + linkSize, block.Location.Y, block.Location.Width, block.Location.Height);
+                    link.Pos += new Vector2(block.Location.Left - (link.Pos.X + linkSize - offset), 0);
```

Comment:
Not sure why this change was made

Reply:
because the block should not move to the right

Reply:
ok so do all blocks move up and down? I thought some of them could move side to side. We might need a Property in movable block that says which directions it can move in